modules/gapi/samples/semantic_segmentation.cpp

```diff
@@ -0,0 +1,133 @@
+#include <opencv2/imgproc.hpp>
+#include <opencv2/gapi/infer/ie.hpp>
+#include <opencv2/gapi/cpu/gcpukernel.hpp>
+#include <opencv2/gapi/streaming/cap.hpp>
+#include <opencv2/highgui.hpp>
+
+const std::string keys =
```

Comment:
This can be `constexpr char[]` if you wish

Reply:
Needs to keep small code style with other samples.

Reply:
OK